pkg/template/helm_template.go

```diff
@@ -9,6 +9,9 @@ import (
 	"io"
 	"io/ioutil"
 	goexec "os/exec"
+	"path"
+
+	"github.com/ghodss/yaml"
```

Comment:
let's not bring in new dependency here. we already are using sigs.k8s.io/yaml (example: https://github.com/vmware-tanzu/carvel-kapp-controller/blob/1844e157b6de4048cec3ba0e53fc699d37e9c71e/pkg/fetch/vendir.go#L17)

Reply:
I Actually chose this one because it was imported in 

https://github.com/vmware-tanzu/carvel-kapp-controller/blob/1844e157b6de4048cec3ba0e53fc699d37e9c71e/pkg/app/app.go#L9

I'll change it to `"sigs.k8s.io/yaml"` as you suggested and maybe in another patch we can work on converging yaml deps?

Reply:
hmm, yeah, we should fix that separately.